node-commons/src/main/java/org/apache/iotdb/commons/auth/role/LocalFileRoleAccessor.java

```diff
@@ -84,10 +84,9 @@ public Role loadRole(String rolename) throws IOException {
         return null;
       }
     }
-
-    try (FileInputStream inputStream = new FileInputStream(roleProfile);
-        DataInputStream dataInputStream =
-            new DataInputStream(new BufferedInputStream(inputStream))) {
+    FileInputStream inputStream = new FileInputStream(roleProfile);
```

Comment:
why do you move the inputstream out side of try? When to close the inputstream?

Reply:
Because FileInputStream will be closed at the same time when dataInputStream close.